ej2-react/code-snippet/tree-view/template-cs2/style.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://cdn.syncfusion.com/ej2/material.css');
+@import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css');
+
+#loader {
+  color: #008cff;
+  height: 40px;
+  left: 45%;
+  position: absolute;
+  top: 45%;
+  width: 30%;
+}
+
+#treeparent {
+  display: block;
+  max-width: 350px;
+  max-height: 350px;
+  margin: auto;
+  overflow: auto;
+  border: 1px solid #dddddd;
+  border-radius: 3px;
+}
+
+.tree-control-wrapper {
+  max-width: 420px;
+  margin: auto;
+  border: 1px solid #dddddd;
+  border-radius: 3px;
+}
+.e-treeview .e-list-text {
+  width: 100%;
+}
+.e-treeview .textcontent {
+  float: left;
+}
+.e-treeview.e-rtl .textcontent {
+  float: right;
+}
+.e-treeview .countcontainer {
+  float: right;
+  margin-right: 5px !important;
+}
+.bootstrap5.e-bigger
+  .template-tree.e-treeview
+  .countcontainer
+  .treeCount.e-badge,
+.bootstrap5-dark.e-bigger
+  .template-tree.e-treeview
+  .countcontainer
+  .treeCount.e-badge {
+  line-height: 11px;
+}
+.template-tree.e-treeview.e-rtl .countcontainer {
+  float: left;
+  margin-left: 5px;
+}
+.material .treeCount.e-badge {
+  vertical-align: middle;
+}
+.bootstrap4 .treeviewdiv .treeCount.e-badge,
```

Comment:
Theme wise styles are not required in UG. Check and update required styles alone @Sureshrajan-18

Reply:
I have removed the Theme wise styles.